tests/integration/test_header_links.py

```diff
@@ -86,7 +86,7 @@ def test_links_not_in_header_when_no_session(self):
         # Then
         self.assert_my_account_link_does_not_exist()
         self.assert_sign_out_link_does_not_exist()
-        self.assert_help_link_exist_not_authenticated()
+        self.assert_help_link_does_not_exist()
```

Comment:
At the moment, we have test for session vs no session. We should have one for no session but still got the cookie.

Reply:
Added a test where session is gone but cookie still exists, thanks!

Reply:
And added one now for social theme as well.